frontend/lib/util/route-util.ts

```diff
@@ -124,6 +124,19 @@ export class RouteMap {
     }
     return false;
   }
+
+  /**
+   * Attempts to get the closest path to the given pathname by
+   * adding or removing a single `/`. If one doesn't exist, this
+   * returns `null`.
+   */
+  getClosest(pathname: string): string | null {
```

Comment:
I'd maybe call this method `slashInsensitive` (as in case insensitive) or `tryWithAndWithoutSlash` to be clearer. `getClosest` sounds to me like we're doing some string matching to guess what the user intended if, for example, they misspelled the path.

Reply:
Good idea!  Done in b0aeb51f.